src/edit.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable import/no-cycle */
```

Comment:
With the ESLint inline comment, we are indirectly changing the rules by disabling these rules, let's fix the codebase instead.